tests/e2e/support/objects/app-admin-settings/spaces/actions.ts

```diff
@@ -77,6 +82,32 @@ export const disableSpace = async (args: {
   ])
 }
 
+export const enableSpace = async (args: {
```

Comment:
these methods(enableSpace, disable, delete, change) are very similar. should they be combined into one? 

But that isn't request to change it :slight_smile:

Reply:
I think we can DRY code here or can't we @SwikritiT

Reply:
@amrita-shrestha @ScharfViktor  yes it's nice to make code DRY, will be in this PR https://github.com/owncloud/web/pull/8629/files